va/va_backend.h

```diff
@@ -529,6 +529,36 @@ struct VADriverContext
 
     char *override_driver_name;
 
+    void *pDisplayContext;
+
+    /**
+     * Error callback.
+     *
+     * The error message will be propagated to the API user through
+     * their error callbacks, or sent to a default output if no callback
+     * is available.
+     *
+     * It is expected that end users will always be able to see these
+     * messages, so they should be used only for serious errors.  For
+     * example, hardware problems or fatal configuration errors.
+     */
+    void (*error_callback)(VADriverContextP pDriverContext,
+                           const char *message);
+    /**
+     * Info callback.
+     *
+     * This has the same behaviour as the error callback, but has its
+     * own set of callbacks to the API user.
+     *
+     * These should be used for informational messages which may be useful
+     * for an application programmer or for debugging.  For example, minor
+     * configuration errors, or information about the reason when another
+     * API call generates an error return.  It is not expected that end
+     * users will see these messages.
+     */
+    void (*info_callback)(VADriverContextP pDriverContext,
+                          const char *message);
+
     unsigned long reserved[41];         /* reserve for future add-ins, decrease the subscript accordingly */
```

Comment:
Please decrease the subscript of the field 'reserved' below for compatibility.

Reply:
There is no release to be compatible with yet.  (The reserved size is presumably going to be reset before the release of libva2?)